tagstudio/src/qt/widgets/preview_panel.py

```diff
@@ -209,6 +222,52 @@ def __init__(self, library: Library, driver: "QtDriver"):
             (self.image_container.size().width(), self.image_container.size().height())
         )
 
+        root_layout = QHBoxLayout(self)
+        root_layout.setContentsMargins(0, 0, 0, 0)
+        root_layout.addWidget(splitter)
+
+    def fill_libs_widget(self, layout: QVBoxLayout):
+        settings = self.driver.settings
+        settings.beginGroup(SettingItems.LIBS_LIST)
+        lib_items = {}
+        for key in settings.allKeys():
+            val = settings.value(key)
+            if len(val) > 45:
+                val = f"{val[0:10]} ... {val[-10:]}"
+            lib_items[key] = val
+        settings.endGroup()
+        new_keys = set(lib_items.keys())
+        if new_keys == self.render_libs:
+            # no need to re-render
+            return
+        self.render_libs = new_keys
+        return self._fill_libs_widget(lib_items, layout)
+
+    def _fill_libs_widget(self, libraries: dict[str, str], layout: QVBoxLayout):
+        # remove any potential previous items
+        for idx in reversed(range(layout.count())):
+            widget = layout.itemAt(idx).widget()
+            layout.removeWidget(widget)
+            # remove from GUI
+            widget.setParent(None)
+
+        label = QLabel("Select a library")
+        label.setAlignment(Qt.AlignCenter)
+        layout.addWidget(label)
+
+        for lib_key, lib_path in libraries.items():
+            button = QPushButton(text=lib_path)
+            button.setObjectName(lib_key)
+
+            def open_library_button_clicked(path):
+                return lambda: self.driver.open_library(path)
+
+            button.clicked.connect(open_library_button_clicked(lib_path))
+            button.setStyleSheet(
+                "background-color: #ffffff; color: #000000; text-align: left; padding-left: 10px;"
+            )
```

Comment:
Proposed style changes to the library section to better match the look and feel of the rest of the program (screenshot includes changes to the section title not featured in the changes below)
![image](https://github.com/TagStudioDev/TagStudio/assets/46939827/22d72345-77a1-4ec6-b1e8-4850debd8369)

```suggestion
            button.setStyleSheet(
                "QPushButton{"
                "background-color:#65000000;"
                "border-radius:6px;"
                "text-align: left;"
                "padding-top: 3px;"
                "padding-left: 6px;"
                "padding-bottom: 4px;"
                "}"
                "QPushButton::hover{background-color:#65AAAAAA;}"
                "QPushButton::pressed{background-color:#65EEEEEE;}"
            )
            button.setCursor(Qt.CursorShape.PointingHandCursor)
```